py/datastore_service.py

```diff
@@ -211,14 +237,16 @@ def initialize_db():
                          'CREATE INDEX index_ids_dates_hours ON segments (segment_id,start_time,start_time_hour,end_time,end_time_hour);' \
                          'CREATE INDEX index_ids_dates_dow ON segments (segment_id,start_time,start_time_dow,end_time,end_time_dow);' \
                          'CREATE INDEX index_ids_dates_hours_dow ON segments (segment_id,start_time,start_time_dow,start_time_hour,end_time,end_time_dow,end_time_hour);')
+          sql_conn.commit()
           sys.stdout.write('Done.' + os.linesep)
           sys.stdout.flush()
       break
     except Exception as e:
       sys.stdout.write('Could not find or create table: ' + repr(e) + os.linesep)
       sys.stdout.flush()
       time.sleep(5)
-
+  if sql_conn:
```

Comment:
this is not needed. the reason i changed everything to `with connect() as sql_conn:` is because that handles closing the connection automatically